components/exact/sources/new-contact-created/new-contact-created.mjs

```diff
@@ -0,0 +1,27 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "exact-new-contact-created",
+  name: "New Contact Created",
+  description: "Emit new event each time a new contact is created. [See the docs](https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=CRMContacts)",
```

Comment:
```suggestion
  description: "Emit a new event each time a new contact is created. [See the docs](https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=CRMContacts)"
```

Reply:
Changed 'Emit new event' to 'Emit a new event' to follow Google's style guide (use articles)